includes/admin/class-wc-admin-addons.php

```diff
@@ -25,11 +25,25 @@ class WC_Admin_Addons {
 	public static function get_featured() {
 		$featured = get_transient( 'wc_addons_featured' );
 		if ( false === $featured ) {
-			$raw_featured = wp_safe_remote_get( 'https://d3t0oesq8995hv.cloudfront.net/add-ons/featured-v2.json', array( 'user-agent' => 'WooCommerce Addons Page' ) );
+			$headers = array();
+			$auth = WC_Helper_Options::get( 'auth' );
+
+			if ( ! empty( $auth['access_token'] ) ) {
+				$headers['Authorization'] = 'Bearer ' . $auth['access_token'];
+			}
+
+			$raw_featured = wp_safe_remote_get(
+				'https://woocommerce.com/wp-json/wccom-extensions/1.0/featured',
+				array(
+					'headers' => $headers,
+					'user-agent' => 'WooCommerce Addons Page',
+				)
+			);
+
 			if ( ! is_wp_error( $raw_featured ) ) {
 				$featured = json_decode( wp_remote_retrieve_body( $raw_featured ) );
 				if ( $featured ) {
-					set_transient( 'wc_addons_featured', $featured, WEEK_IN_SECONDS );
+					set_transient( 'wc_addons_featured', $featured, DAY_IN_SECONDS );
```

Comment:
@mattyza do you think that it’s ok to keep caching featured page content, or should we stop caching it?
Caching reduces the number of requests to WooCommerce.com, but it prevents us from making immediate changes.

Reply:
Perhaps a job for another day, but it seems to me like the optimal approach would be:

1. Continue to cache for performance reasons, as well to increase the likelihood of seeing content if for some reason the site can't connect to WCCOM
2. Clear this cache on installation/uninstallation/activation/deactivation of a plugin, resync of connection to WCCOM, etc.: this way, if a customer buys a new plugin they're likely to see the same recommendations... but then _if they install it_ their recommendations will get updated!

Reply:
This makes sense @Dan-Q. Leaving the caching to last a day will keep the number of requests small but it will still allow us to change Algolia rules and to know that the changes will apply within 24 hours.

To keep cache clearing simple I suggest that we just add it to the `subscriptions` helper API call https://github.com/Automattic/woocommerce.com/blob/475ada19df47a07ac057a4fa1c207d129be1eb8a/plugins/woocommerce/includes/admin/helper/class-wc-helper.php#L1264

Reply:
If we decide to make any changes, can we do them in a separate PR? To avoid delaying this PR.

Reply:
Caching definitely makes sense, here. As Dan mentioned, having a way to invalidate this cache on demand is important. This can be done on disconnecting and reconnecting to WooCommerce.com, which would give Happiness Engineers a way to instruct customers to refresh their cache if needed.

There are, I'm sure, other options which we could explore in future PRs.

Caching for 1 day seems reasonable to me.